CONTRIBUTING.md

```diff
@@ -358,9 +358,30 @@ separated by a horizontal rule `---`,
 where `#` is to be replaced with an increasing
 and per-page unique number.
 
-**Note**: The footnote text at the bottom of the page
+<div class="infobox"><p>
+The footnote text at the bottom of the page
 must all be written one line per footnote.
-Line breaks **cannot** be used.
+Line breaks <strong>cannot</strong> be used.
+</p></div>
+
+
+### Info boxes
+
+Info boxes can be used similarly to footnotes,
+but for information that is important for the reader to understand
+before continuing to read the rest of the page.
+
+The current syntax uses in-line HTML to render these paragraphs with a
+different CSS style.
+These paragraphs must be separated with a blank line above and below similar
+to images or code blocks.
+
+```md
+<div class="infobox"><p>
+Text here as usual, using semantic linefeed rules.
+If you need text-formatting, you <strong>must</strong> use in-line HTML.
```

Comment:
I believe you can use in-line Markdown again if you have a blank line after the opening sequence of `<div … p>` – or at least that's how it works in Github's parser. Please test first.

Reply:
Doesn't seem to work. Still displays `*text*` as `*text*` and not *text*.

Reply:
Sad!